docset/winserver2022-ps/netwnv/Get-NetVirtualizationProviderAddress.md

```diff
@@ -139,12 +153,14 @@ Accept wildcard characters: False
 ```
 
 ### -ManagedByCluster
-Specifies an array of Boolean values that determines whether a highly available Provider Address configuration entry exists on a Hyper-V Network Virtualization gateway.
-If you use a Provider Address cluster resource, this value is $True; otherwise, the value is $False.
-This parameter is a read-only parameter, used by an administrator to diagnose problems.
+
+Specifies an array of Boolean values that determines whether a highly available Provider Address
+configuration entry exists on a Hyper-V Network Virtualization gateway. If you use a Provider
+Address cluster resource, this value is $True; otherwise, the value is $False. This parameter is a
+read-only parameter, used by an administrator to diagnose problems.
```

Comment:
Minor changes: bold the type name, downcase and code-format the boolean variables.
```suggestion
Specifies an array of **Boolean** values that determines whether a highly available Provider
Address configuration entry exists on a Hyper-V Network Virtualization gateway. If you use a
Provider Address cluster resource, this value is `$true`. Otherwise, the value is `$false`. This
parameter is a read-only parameter, used by an administrator to diagnose problems.
```